tests/cross_functional/scale/noobaa/test_delete_objects.py

```diff
@@ -190,3 +202,122 @@ def test_delete_objects(
         # stop the io running in the background
         event.set()
         io_thread.result()
+
+    @pytest.fixture(scope="session")
+    def create_bucket_verify_object_deletion(
+        self,
+        request,
+        reduce_expiration_interval_session,
+        change_lifecycle_schedule_min_session,
```

Comment:
Do we really need to use the `session` scope fixtures do adjust MCG's background workers configuration?

This means that other tests might get affected by these conditions we only used as a workaround for testing expiration in a reasonable time.

Reply:
1. The reason why i used session scoped fixture here is because i have a session scoped fixture that is used to create and verify object expiration. If I use function scoped fixture within the session scope fixture, it's not really possible to call function scoped within session scope fixture. if I use function scope fixture in the test itself, then the expiration intervals are reset back to default which will affect the session scoped fixture of object expiration verification and it is not useful for us. Hope I didnt confuse you 😃 

2.  I dont think it will affect any other tests as this test will not be executed part of other test suites. This will only get executed when we call `scale_long` marker